htdocs/projet/tasks.php

```diff
@@ -200,6 +200,19 @@
 			$action='create';
 			$error++;
 		}
+		
```

Comment:
Your patch add code, sql check but does not guarantee that creation is done just after by another user.

Adding 
$action = 'create' on line 248 (261) should be enough and work even on intensive tasks creations.

Reply:
yes, but when the form is proposed again, the reference is already updated automatically, so user just need to click again on save button.